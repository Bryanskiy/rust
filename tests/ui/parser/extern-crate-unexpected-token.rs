extern crte foo; //~ ERROR expected one of `crate`, `dyn`, or `{`, found `crte`
