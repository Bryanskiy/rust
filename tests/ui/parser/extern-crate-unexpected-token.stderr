error: expected one of `crate`, `dyn`, or `{`, found `crte`
  --> $DIR/extern-crate-unexpected-token.rs:1:8
   |
LL | extern crte foo;
   |        ^^^^ expected one of `crate`, `dyn`, or `{`
   |
help: there is a keyword `crate` with a similar name
   |
LL | extern crate foo;
   |        ~~~~~

error: aborting due to 1 previous error

